pages/application.tsx

```diff
@@ -1,7 +1,87 @@
-import React, { ReactElement } from "react";
-
+import React, { ReactElement } from 'react'
+import {
+  QuestionType,
+  ShortText,
+  LongText,
+  Dropdown,
+  Checkboxes,
+} from '../common/types'
+import { EXAMPLE_QUESTIONS } from '../common/constants'
+// export async function getStaticProps() {
+//   return {
+//     props: { EXAMPLE_QUESTIONS }
+//   }
+// }
+const shortText = (q: ShortText) => {
```

Comment:
for component names, usually I do uppercase. thoughts?

Reply:
I have no preference so that sounds good - I'll fix it!